packages/telemed-ehr/app/src/telemed/features/telemed-admin/EditState.tsx

```diff
@@ -39,12 +40,13 @@ export default function EditStatePage(): JSX.Element {
     {
       onSuccess: (location) => {
         setIsOperateInStateChecked(Boolean(location && location.status === 'active'));
+        setStateName(location?.name);
       },
     },
   );
 
   const mutation = useMutation(({ location, newStatus }: { location: Location; newStatus: string }) =>
-    updateStateLocationStatus(fhirClient, location, newStatus),
+    updateStateLocationStatusAndName(fhirClient, location, newStatus, stateName ?? ''),
```

Comment:
I think it would be better if we have a validation for the `stateName` variable. Updating stateName as an empty string wouldn't be the best option here.